deploy/docker/README.md

```diff
@@ -0,0 +1,18 @@
+######################
+
+### pump.io http://pump.io Docker image - listens port 443
+
+1) Copy pump.io.json.sample to pump.io.json and configure pump.io.json according to the needs
```

Comment:
Hmm, okay. pump.io doesn't currently accept configuration through env variables but there are plans to... I can reprioritize those. See #1202